### DISCUSSION TITLE
[1363080]庄园领主{本体}

### DISCUSSION THREAD

**Discussion by Temlpe** [pjy612/SteamManifestCache#622] [Share]
### Steam 游戏ID

1363080

### 密钥列表

```hash
228988;1845444d5e2cfd0ae65ae4a8fedb6e2fbf776fcc5b913ab4ac461bc9a74f8358
228990;44d8c45ce229a11c4f231a3d2a350eaf80b0d69a8af938ec7ccca720f694b0e8
229007;eba9fa9ff4f811929c119074450f49f739113bcae9317ded9bea262aa21ec58d
229033;9a789440bbc16131547ac6c062e5f168f7684da533a2899a71fdfaaeec21efc3
1363081;28763f0001125b10d1d269c0655d8d3f9e6a7678dd9fed0af3f4a37701443e70
```

### 清单附件

[229007_4477590687906973371.zip](https://github.com/pjy612/SteamManifestCache/files/15131729/229007_4477590687906973371.zip)

### 备注

PS：大佬你的那个DepotDownloader[魔改]的项目能更新一下吗，用这个获取清单挺方便的
code:79E3D7BFF9207F05
email:[uppuyxw@foxmail.com](mailto:uppuyxw@foxmail.com)

**Comment by pjy612** [comment]
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/c07e432e-d4ec-470b-9977-b10aba7ce470)
Shared Install 是免费的 不用分享
